app/models/user.rb

```diff
@@ -5,10 +5,16 @@ class User < ActiveRecord::Base
 
   has_many :recommended_items
   has_many :television_shows, through: :recommended_items
-
   has_many :following_relationships,
     class_name: "Relationship", foreign_key: :follower_id
   has_many :following, through: :following_relationships, source: :followed
+  has_many :follower_relationships,
+    class_name: "Relationship", foreign_key: :followed_id
+  has_many :followers, through: :follower_relationships, foreign_key: :follower
+
+  def self.recommended_to_follow(except:)
```

Comment:
Naming ideas welcome. I originally called this `recommended`, but I think the terminology is blurry given that this is a recommendation app. My other idea was `influencers`, like "people you should follow because they have good taste" (even though the algorithm for finding influencers right now is As Dumb As Possible).